ui/lib/pki/addon/components/page/pki-configuration-edit.hbs

```diff
@@ -0,0 +1,98 @@
+      <h2 class="title is-size-5 has-border-bottom-light page-header">
+        Global URLs
+      </h2>
+      {{#if @urls.canSet}}
+        {{#each @urls.allFields as |attr|}}
+          <FormField @attr={{attr}} @model={{@urls}} @showHelpText={{false}} />
+        {{/each}}
+      {{else}}
+        <EmptyState
+          class="is-box-shadowless"
+          @title="You do not have permission to set URLs"
+          @message="Ask your administrator if you think you should have access to:"
+        >
+          <code>POST /{{@backend}}/config/urls</code>
+        </EmptyState>
+      {{/if}}
+    </fieldset>
+
+    <fieldset class="box is-shadowless is-marginless is-borderless is-fullwidth" data-test-crl-edit-section>
+      <h2 class="title is-size-5 has-border-bottom-light page-header">
+        Certificate Revocation List (CRL)
+      </h2>
+      {{#if @crl.canSet}}
+        {{#each @crl.formFields as |attr|}}
+          {{#if (eq attr.name "ocspExpiry")}}
+            <h2 class="title is-size-5 has-border-bottom-light page-header">
+              Online Certificate Status Protocol (OCSP)
+            </h2>
+          {{/if}}
+          {{#if (or (includes attr.name this.alwaysRender) (not @crl.disable))}}
+            {{#let (get @crl attr.options.mapToBoolean) as |booleanValue|}}
+              <div class="field">
+                <TtlPicker
+                  data-test-input={{attr.name}}
+                  @onChange={{fn this.handleTtl attr}}
+                  @label={{attr.options.label}}
+                  @labelDisabled={{attr.options.labelDisabled}}
+                  @helperTextDisabled={{attr.options.helperTextDisabled}}
+                  @helperTextEnabled={{attr.options.helperTextEnabled}}
+                  @initialEnabled={{if attr.options.isOppositeValue (not booleanValue) booleanValue}}
+                  @initialValue={{get @crl attr.name}}
+                />
+              </div>
+            {{/let}}
+          {{/if}}
+        {{/each}}
+      {{else}}
+        <EmptyState
+          class="is-box-shadowless"
+          @title="You do not have permission to set revocation configuration"
+          @message="Ask your administrator if you think you should have access to:"
+        >
+          <code>POST /{{@backend}}/config/crl</code>
+        </EmptyState>
+      {{/if}}
+    </fieldset>
+
+    <div class="field is-grouped box is-fullwidth is-bottomless">
+      <div class="control">
+        {{#if (or @urls.canSet @crl.canSet)}}
+          <button
+            type="submit"
+            class="button is-primary {{if this.save.isRunning 'is-loading'}}"
+            disabled={{this.save.isRunning}}
+            data-test-configuration-edit-save
+          >
+            Save
+          </button>
+        {{/if}}
+        <button
+          {{on "click" this.cancel}}
+          type="button"
+          class="button has-left-margin-s"
```

Comment:
when the form is submitting or `this.save.isRunning`, do we want to disable the button?

Reply:
thank you!